harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/geoPREST/Harvester.java

```diff
@@ -262,6 +265,33 @@ private RecordInfo getRecordInfo(Element record) {
 
     }
 
+    protected Date parseDateJdk11(String pubDate) throws ParseException {
+        Locale wellKnownLocales[] = {Locale.ENGLISH, Locale.FRENCH, Locale.GERMAN, Locale.ITALIAN};
```

Comment:
Use the array indicators in the type, not in the variable name:
```suggestion
        Locale[] wellKnownLocales = {Locale.ENGLISH, Locale.FRENCH, Locale.GERMAN, Locale.ITALIAN};
```